prow/config.yaml

```diff
@@ -62,18 +62,17 @@ presubmits:
     - master
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/bazelbuild:0.11
+      - image: gcr.io/k8s-testimages/bazelbuild:v20170720-7762904d
         args:
-        - "--pull=$(PULL_REFS)"
+        - "--repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)"
```

Comment:
I'm not sure I understand the benefit of adding additional boilerplate to the prow configuration. (Basically every job is now adding the exact same two arguments, rather than having it specified once in the image.)

All of the prow-native test images use the env-var based pattern right now, so this PR is making things less consistent.